### DISCUSSION TITLE
Narrow types on `QInput.nativeEl` and `QInput.getNativeElement`

### DISCUSSION THREAD

**Discussion by mv-go** [quasarframework/quasar#15121] [Ideas / Proposals] [answered]
According to [docs](https://quasar.dev/vue-components/input#qinput-api) the type of `nativeEl` and the return type of `getNativeElement` is
```The native input/textarea DOM Element```

However, the typings that shipped with the library suggest that the type here is a very general `Element`.

Could it be possible to narrow it down to `HTMLInputElement | HTMLTextAreaElement`? Or are there some edge cases that I'm missing?

**Comment by mv-go** [comment]
This could actually qualify as a `bug`, but I've decided to start here in case I'm missing something :)

**Comment by yusufkandemir** [comment, marked as answer]
Good idea 👍 It will be addressed by #15129.

Released in https://github.com/quasarframework/quasar/releases/tag/quasar-v2.11.0
